Modules/ITS/itsEPN.json

```diff
@@ -0,0 +1,127 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks": {
+      "ITSClusterTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::its::ITSClusterTask",
+        "moduleName": "QcITS",
+        "detectorName": "ITS",
+        "cycleDurationSeconds": "180",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "compclus"
+        },
+        "location": "remote",
+        "taskParameters": {
+          "layer": "1111111",
+          "clusterDictionaryPath": "/home/epn/odc/files/ITSdictionary.bin",
+          "runNumberPath": "/home/its/QC/workdir/infiles/RunNumber.dat",
+          "geomPath": "/home/epn/odc/files/o2sim_geometry.root",
```

Comment:
is it fine that they are in different homes ?

Reply:
Yes, it's fine. The `runNumberPath` will become obsolete and be removed soon. We will fetch it from ECS then. Thank you.